Homeword submission/Page2.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<title>Relaxr</title>
+		<link href="./normalize.css" rel="stylesheet"/>
+		<link href="./stylesheet2.css" rel="stylesheet"/>
+	</head>
+	<body>
+		<div class="container">
+			<header>
+				<div class="header">
+					<h2 class="relax">Relaxr</h2>
+					<nav class="navhead">
+						<a href="#">About</a>
+						<a href="#">FAQ</a>
+						<a href="#">Team</a>
+						<a href="#">Contact Us</a>
+						<a href="#">Blog</a>
+					</nav>
+				</div>
+			</header>
+			<section>
+				<div class="body1">
+					<div class="pic">
+						<h3>How I implemented Relaxr in 2 weeks and changed my life.</h3>
+						<img src="blog_photo1.jpg" alt="Blog1" style="width:595px;height:650px:">
+						<p><span class="biggray">L</span>orem ipsum dolor sit amet, consectetur adipisicing elit. Possimus debitis distinctio assumenda, quaerat incidunt ullam labore nisi odit eligendi ab ad quasi vero, eaque vel odio sed sit corporis! Voluptatem! Lorem ipsum dolor sit amet, consectetur adipisicing elit. Reprehenderit optio unde quam nulla quisquam, et pariatur aspernatur consectetur sint nisi dolor dolores non debitis inventore ipsum esse ullam assumenda illum!</p>
```

Comment:
This should have been achieved by using a pseudo class.

That is 

```
p :first-letter{
    font-size: 50px;
}
```

Notice how I target the whole paragraph